tests/integration_tests/tests/agentless_tests/component/test_component_cascading.py

```diff
@@ -27,12 +29,17 @@
     generate_scheduled_for_date)
 
 
+@pytest.mark.usefixtures('testmockoperations_plugin')
 class ComponentCascadingCancelAndResume(AgentlessTestCase):
     def _wait_for_component_install(self, deployment_id):
         # waiting for component to create it's deployment
-        do_retries(verify_deployment_env_created,
-                   30,
-                   deployment_id=deployment_id)
+        do_retries(
```

Comment:
this is here many times, package this into a method?

Reply:
Done and updated